src/Umbraco.Web/umbraco.presentation/umbraco/dialogs/sort.aspx.cs

```diff
@@ -154,7 +154,7 @@ private static SortableNode CreateNode(string id, int sortOrder, string name, Da
         public struct SortableNode
         {
             public string id;
-            public int sortOder;
+            public int sortOrder;
             public string Name;
             public string icon;
             public DateTime createDate;

```

Comment:
It might be worth to name to public properties: Id, Name, Icon, CreateDate and SortOrder.